plugins/paymethod/paypal/locale/cs_CZ/locale.xml

```diff
@@ -4,8 +4,8 @@
 <!--
   * plugins/paymethod/paypal/locale/cs_CZ/locale.xml
   *
-  * Copyright (c) 2014-2016 Simon Fraser University Library
-  * Copyright (c) 2006-2014 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2006-2009 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
```

Comment:
I've maintained the en_US copyright dates reasonably carefully; it's most likely that this was changed by a 3rd party. It's best to stay consistent with what en_US has.